synapse/models/infotech.py

```diff
@@ -594,8 +600,14 @@ def getModelDefs(self):
                 }),
                 ('it:cmd', ('str', {'strip': True}), {
                     'doc': 'A unique command-line string.',
-                    'ex': 'foo.exe --dostuff bar',
-                }),
+                    'ex': 'foo.exe --dostuff bar'}),
+
+                ('it:cmd:session', ('guid', {}), {
+                    'doc': 'A command line session with multiple commands run over time.'}),
+
+                ('it:cmd:history', ('guid', {}), {
+                    'doc': 'A single command executed within a session.'}),
```

Comment:
Just want to confirm that history is meant to only capture a single command, vs X number of previous commands?